packages/ilios-common/addon-test-support/ilios-common/page-objects/components/editable-field.js

```diff
@@ -0,0 +1,32 @@
+import {
+  clickable,
+  count,
+  create,
+  isVisible,
+  property,
+  text,
+  triggerable,
+} from 'ember-cli-page-object';
+import { hasFocus } from 'ilios-common';
+import fadeText from './fade-text.js';
+
+const definition = {
+  scope: '[data-test-editable-field]',
+  value: text(),
+  hasEditIcon: isVisible('[data-test-edit-icon]'),
+  editable: {
+    scope: '[data-test-edit]',
+    edit: clickable(),
+    iconCount: count(),
+    title: property('title'),
```

Comment:
I don't know that it makes a huge different, but this should probably pull from [attribute](http://ember-cli-page-object.js.org/docs/v2.3.x/api/attribute) instead of `property`.